roles/core/dhcp_server/readme.rst

```diff
@@ -116,6 +116,7 @@ Files generated:
 Changelog
 ^^^^^^^^^
 
+* 1.0.8: Add unitary molecule tests. Osmocl <osmocl@osmo.cl>
```

Comment:
Please don't change the role version when there is no functional change or documentation improvement.

Reply:
Ok I will remove it.

Reply:
Corrected in commit: fd4d355